scratchapi2/excs.py

```diff
@@ -1,9 +1,5 @@
-""" Scratch API Error """
+"""Scratch API errors"""
 
 class ScratchAPIError(Exception):
-    """ Scratch API Error (Generic) """
-    pass
-
-class RemovedFeatureError(NotImplementedError):

```

Comment:
I saw that this was used for the "projects" dummy submodule. It's better to just completely remove the submodule, especially since you haven't released it to PyPi yet.